src/FluentResults/Factories/Results.cs

```diff
@@ -93,12 +93,12 @@ public static Result Ok()
         }
         
         /// <summary>
-        /// Creates a failed result with the given error
+        /// Creates a failed result with the given errors
         /// </summary>
-        public static Result Fail(Error error)
+        public static Result Fail(params Error[] errors)
```

Comment:
I would keep the method with a single error so that the clients know better that also one single error is enougth. But we can also add the params method style.

Reply:
Ok, I return method with single error.